File: Shared/Students.Models/ReferenceModels/FEAProgram.cs
namespace Students.Models.ReferenceModels;

/// <summary>
/// ВЭД программа.
/// </summary>
public class FEAProgram
{
  /// <summary>
  /// Идентификатор ВЭД.
  /// </summary>
  public Guid Id { get; set; }

  /// <summary>
  /// Имя ВЭД.
  /// </summary>
  public string? Name { get; set; }
}

File: Client/src/storage/catalogConfigs/feaProgram.js
import {
    useGetAllAsync,
    useGetAllPagedAsync,
    useGetOneByIdAsync,
    useAddOneAsync,
    useEditOneAsync,
    useRemoveOneAsync,
} from '../crud/feaProgramCrud.js';
import  feaProgramModel   from '../models/feaProgram.js';

export default {
    detailsLink: 'feaProgram',
    hasDetailsPage: false,
    serverPaged: false,
    properties: feaProgramModel,
    crud: {
        useGetAllAsync,
        useGetAllPagedAsync,
        useGetOneByIdAsync,
        useAddOneAsync,
        useEditOneAsync,
        useRemoveOneAsync,
    },
    columns: [
        {
            title: 'ВЭД программа',
            dataIndex: 'name',
            key: 'name',
        },
    ],
    dataConverter: (data) => data,
};